### DISCUSSION TITLE
GPU Particle system is supported?

### DISCUSSION THREAD

**Discussion by orosbogdan** [MonoGame-Extended/Monogame-Extended#942] [General]
As the title says, is it supported ?

**Comment by Ultra4life** [comment]
You can check my post here : 
https://github.com/craftworkgames/MonoGame.Extended/issues/954

Particle Documentation is Outdated and Code Example No Longer Works in MonoGameExtended Version 4.0.3

For thoses who are  stuck like me :
I had to Downgrade MonoGameExtended to version 3.8.0
and re-install MonoGame.Extended.Particules  3.8.0, using the Dotnet commands, to finally make it work

`dotnet add package MonoGame.Extended --version 3.8.0`
`dotnet add package MonoGame.Extended.Particles --version 3.8.0`

**Comment by AristurtleDev** [comment]
@orosbogdan and @Ultra4life, the particle system was recently completely overhauled and updated in the 5.2.0 release.  This included fully updating the documentation as well on the website.  You can view the documentation at https://www.monogameextended.net/docs/features/particles/quick_start/
